examples/monaco-graphql-webpack/package.json

```diff
@@ -12,17 +12,19 @@
   },
   "dependencies": {
     "graphql": "14.6.0",
-    "monaco-graphql": "file:../../packages/graphiql",
+    "monaco-graphql": "file:../../packages/monaco-graphql",
     "prettier": "^2.0.2"
   },
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/preset-env": "7.9.5",
     "@types/prettier": "^2.0.0",
+    "acorn": "^8.0.3",
```

Comment:
I ran `yarn` both in the root as well as the monaco example folder. I got the error saying that `acorn` path is missing and that it needs to be installed.